plugin/mac/setup.go

```diff
@@ -0,0 +1,27 @@
+package mac
+
+import (
+	"github.com/coredns/coredns/core/dnsserver"
+	"github.com/coredns/coredns/plugin"
+	"github.com/mholt/caddy"
+)
+
+func init() {
+	caddy.RegisterPlugin("mac", caddy.Plugin{
+		ServerType: "dns",
+		Action:     setup,
+	})
+}
+
+func setup(c *caddy.Controller) error {
+	c.Next()
+	if c.NextArg() {
```

Comment:
I think we have a way to avoid several time the same "mac" plugin in the stanza

Reply:
I didn't get the point if this comment. Could you clarify?

Reply:
I don't think that we have a valid, tested way to do it now. It is TODO in case of atepcli

Reply:
See sample in cache plugin:

```
func cacheParse(c *caddy.Controller) (*Cache, error) {
	ca := New()

	j := 0
	for c.Next() {
		if j > 0 {
			return nil, plugin.ErrOnce
		}
		j++
              ....
```